.github/workflows/test_workflow_scripts/golang-docker.sh

```diff
@@ -61,8 +61,8 @@ send_request(){
     wait
 }
 
+container_name="ginApp_${i}"
 for i in {1..2}; do
-    container_name="ginApp_${i}"
     send_request &
     sudo -E env PATH=$PATH ./../../keployv2 record -c "docker run -p8080:8080 --net keploy-network --rm --name $container_name gin-mongo" --container-name "$container_name" --generateGithubActions=false &> "${container_name}.txt"
```

Comment:
wouldn't this move it out of loop ? like for different container name  it was loke ginApp_1,ginApp_2..now it would be ginApp_ as I isn't defined